test/integration/TestDataProvider.js

```diff
@@ -8,12 +8,17 @@ module.exports = class TestDataProvider {
                 {
                     name: 'testname Translation One',
                     description: 'Translation One testDescription',
-                    locale_id: 1,
+                    locale_id: 1, // de-ch from fixture
```

Comment:
Hmm, this is not necessarily id 1..... I think these tests are likely to break on other systems (e.g. if someone ran the migrations on the language-service multiple times and did not reset the index of the locale table).

Reply:
Actually I had the same in mind when I was looking at this but the locales are not loaded form the language service, they are provided as json fixture for the locale mode which uses the memory connector as a datasource.
https://github.com/joinbox/loopback-component-translations/blob/master/src/fixtures/locales.json
https://github.com/joinbox/loopback-component-translations/blob/master/test/support/server/datasources.json